pkg/admission/test/integ/admission_integ_test.go

```diff
@@ -256,19 +259,33 @@ var _ = Describe("Admission server integration tests", func() {
 					Namespace: namespace,
 				},
 			}
+			// Set the default resource explicitly to be noobaa-default-backing-store
+			// of the account that run these tests (admin@noobaa.io) before starting the first delete operation test.
+			// We use an RPC call update_account_s3_access
+			// https://github.com/noobaa/noobaa-operator/issues/1045
+			nbClient := system.GetNBClient()
+			err := nbClient.UpdateAccountS3Access(nb.UpdateAccountS3AccessParams{
+				Email:           "admin@noobaa.io",
+				S3Access:        true,
+				DefaultResource: &defaultBs.Name,
+			})
+			if err != nil {
+				fmt.Printf("setting the default resource created an error %s\n", err)
+			}
 
 			// try to delete noobaa-default-backing-store and failing because it has data buckets
 			result, err = KubeDelete(defaultBs)
 			Expect(result).To(BeFalse())
 			Ω(err).Should(HaveOccurred())
 			Expect(err.Error()).To(Equal("admission webhook \"admissionwebhook.noobaa.io\" denied the request: cannot complete because pool \"noobaa-default-backing-store\" in \"IN_USE\" state"))
 		})
-		It("Should Allow", func() {
+		It("Should Allow bs", func() {
```

Comment:
while at it, maybe update the message to something clearer - e.g.: `Should Allow backingstore deletion`

Reply:
I changed it. The way this file was written was in a hierarchical description, so now it will be:
`Delete operations -> Should Allow backingstore deletion` (mentioning twice the deletion). it is not constant with the rest of the tests, but I agree it will be more maintainable and readable.